app/code/Magento/Catalog/Controller/Adminhtml/Product/Initialization/Helper.php

```diff
@@ -317,6 +318,25 @@ protected function normalize(array $productData)
         return $productData;
     }
 
+    /**
+     * Internal trim data
+     *
+     * @param array $productData
+     * @return array
+     */
+    protected function trimData(array $productData)
```

Comment:
This is a copy of `normalize` method which is going to be removed: https://github.com/birmo/magento2/blob/2.2-develop/app/code/Magento/Catalog/Controller/Adminhtml/Product/Initialization/Helper.php#L303

Need to be found out in favor of what `normalize` is going to be removed.